### ISSUE TITLE
zones -> /srv/salt/zones broken link

### ISSUE THREAD

**Issue by ogabrielsantos** [saltstack-formulas/bind-formula#37]
Why `bind/zones` a symbolic link?
Why it is hardcoded without possibility of custom path?

What about a `root_dir` pointing to `~/workspace/servers/` instead of `/srv/salt`?
What about `file_roots` other than default `base` setup?

````
file_roots:
  dev:
    - /srv/salt/dev/states
    - /srv/salt/prod/states
  prod:
    - /srv/salt/prod/states
````

Currently I need to manual change `bind-formula` in order to make things work.

Maybe add another `file_root` entry pointing to my custom bind directory can work but this seems a hack instead of a default expected approach:

````
file_roots:
  dev:
    - /srv/salt/prod/states/files/var/
    - /srv/salt/dev/states
    - /srv/salt/prod/states
  prod:
    - /srv/salt/prod/states/files/var/
    - /srv/salt/prod/states

var/
└── bind/
    └── hosts
       ├── zone-file1.local
       ├── zone-file2.local
````

EDIT: Hacky symlinks will be a issue [because master stop responding when I've specifics symlinks](https://github.com/saltstack/salt/issues/27856)

tks (:

**Comment by ogabrielsantos** [comment]
Maybe mapping `zones_source_dir` default to `/srv/salt/zones` inside `map.jinja`,  so user can replace inside their own pillars, will be a fine approach.

**Comment by cybacolt** [comment]
@ogabrielsantos, this is bugging me too - if its ok, i'll do a PR for it, as you described using map.jinja.

**Comment by ogabrielsantos** [comment]
@cybacolt I've already updated here https://github.com/ogabrielsantos/bind-formula/blob/master/bind/map.jinja#L6. I can merge saltstack-formulas/bind-formula back and do a PR if you want.

**Comment by cybacolt** [comment]
@ogabrielsantos too quick :)

yeah, a PR back to base would be good. i'll stop my changes now.

**Comment by ogabrielsantos** [comment]
@cybacolt I took the liberty of include looping fix and file names conflicts (I think this is essential to do this formula run as it said that will work initially - and because [Some improvements #28](https://github.com/saltstack-formulas/bind-formula/pull/28) is a 5 yo PR that discusses fixes and features, delaying the merge fix.

**Comment by cybacolt** [comment]
@ogabrielsantos working around this with my own symlink for now. look forward to seeing it in master some time soon!

**Comment by babilen** [comment]
This was closed by #42
